### ISSUE TITLE
Checkbox/radio focus not visible in Windows 10 High Contrast mode

### ISSUE THREAD

**Issue by ankz2390** [angular/material2#13227]
Bug, feature request, or proposal:
Bug

What is the expected behavior?
Focus should be visible on mat-checkbox on High contrast mode when you tab into the checkbox

What is the current behavior?
No focus on checkbox

 What are the steps to reproduce?
Tab into a checkbox in windows high contrast mode

What is the use-case or motivation for changing an existing behavior?
Catering for accessibility for disabilities requiring high contrast modes

Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Edge
